core/src/filed/fd_plugins.cc

```diff
@@ -1140,7 +1141,15 @@ bool PluginNameStream(JobControlRecord* jcr, char* name)
       Dmsg2(debuglevel, "End plugin data plugin=%p ctx=%p\n", plugin,
             jcr->plugin_ctx);
       if (b_ctx->restoreFileStarted) {
-        PlugFunc(plugin)->endRestoreFile(jcr->plugin_ctx);
+        /* PlugFunc(plugin)->endRestoreFile(jcr->plugin_ctx); */
+        bRC ret = PlugFunc(plugin)->endRestoreFile(jcr->plugin_ctx);
+        Dmsg1(0, "endRestoreFile ret: %d\n", ret);
+        if (ret < 0) {
```

Comment:
According to the bRC enum in lib/plugins.h this value cannot be less than zero.

Reply:
Changed as suggested.